src/containers/MainPage/MainPage.js

```diff
@@ -13,9 +12,8 @@ class MainPage extends Component {
     super(props);
     this.state = {
       query: '',
-      noResult: false,
+      errors: {},
     };
-    this.onInputChange = this.onInputChange.bind(this);
```

Comment:
You can directly use arrow functions no need to bind class methods

Reply:
Done.